dom-querying/main.js

```diff
@@ -0,0 +1,14 @@
+console.log('hello, world');
+var q = document.querySelector('h1');
+console.log('h1 query selectors:' + ' ' + q);
+console.dir('h1 directory query' + ' ' + q);
+var p = document.querySelector('#explanation');
+console.log(' #explanation id query selector' + ' ' + p);
+console.dir('p directory query ' + ' ' + p);
+var f = document.querySelector('.hint');
+console.log('hint class query selector ' + ' ' + f);
+console.dir('hint class query directory' + ' ' + f);
```

Comment:
For `console.dir`, only pass the variable with no label.

Reply:
![Screen Shot 2021-02-02 at 11 00 35 AM](https://user-images.githubusercontent.com/69738204/106626920-35395d00-6546-11eb-8ccb-9707f4349423.png)
![Screen Shot 2021-02-02 at 11 00 43 AM](https://user-images.githubusercontent.com/69738204/106626930-38344d80-6546-11eb-8e6f-8d38c7cd4bab.png)